project/Settings.scala

```diff
@@ -60,7 +60,7 @@ object Settings {
 
     // Pure JS libraries
     val fomanticUI = "2.8.7"
-    val ocsVersion = "2021102.1.1"
+    val ocsVersion = "2021101.2.2"
```

Comment:
This was my bad, I used the wrong version number when I published the artifacts earlier